### PR TITLE
Feat/update dapp

### PR DESCRIPTION
**Pull request by aminlatifi**
#274

### CODE REVIEW

**Comment by aminlatifi** [comment]
Refs #262

File: src/pages/contribute/components/Welcome.js [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
 
 const Comp = ({ effectiveBalance }) => {
+  const { isReady } = useContext(OnboardContext);
+  const [showPendingScore, setShowPendingScore] = useState(false);
+
+  React.useEffect(() => {
+    if (isReady) {
+      setShowPendingScore(effectiveBalance > 0);
+    }
+  }, [isReady, showPendingScore, effectiveBalance]);
```
Comment (aminlatifi) [inline src/pages/contribute/components/Welcome.js:16]: `showPendingScore` should be removed from dependencies

**Review by aminlatifi — COMMENTED** [review]
I don't know much about variables meaning in the code (effective balance I mean). But all the changes made to getting pending balance and user is whitelisted seems OK

File: src/pages/contribute/components/Welcome.js (hunk 1 above)
Comment (MerlinEgalite) [reply src/pages/contribute/components/Welcome.js]: done ✅

**Review by MerlinEgalite — COMMENTED** [review]
thx for comments I updated the code

**Pull request merged**